incubator/hydra/templates/configmap.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "hydra.fullname" . }}
+  labels:
+    app.kubernetes.io/name: {{ include "hydra.name" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    helm.sh/chart: {{ include "hydra.chart" . }}
+data:
+  config.yaml: |
+    {{ toYaml .Values.config }}
```

Comment:
This can produce an invalid configmap, maybe using `{{ toYaml .Values.config  | indent 4}}` would be better?

Reply:
You're correct @Demonsthere, thanks for the suggestion!